pipeline/outputs/loki.md

```diff
@@ -10,28 +10,29 @@ Be aware there is a separate Golang output plugin provided by [Grafana](https://
+| drop\_single\_key               | If set to true and after extracting labels only a single key remains, the log line sent to Loki will be the value of that key in line\_format. If set to `raw` and the log line is a string, the log line will be sent unquoted.                                                                                                                                                                                                | off               |
+| line\_format                    | Format to use when flattening the record to a log line. Valid values are `json` or `key_value`. If set to `json`,  the log line sent to Loki will be the Fluent Bit record dumped as JSON. If set to `key_value`, the log line will be each item in the record concatenated together \(separated by a single space\) in the format.                                                                                             | json              |
+| auto\_kubernetes\_labels        | If set to true, it will add all Kubernetes labels to the Stream labels                                                                                                                                                                                                                                                                                                                                                          | off               |
+| tenant\_id\_key                 | Specify the name of the key from the original record that contains the Tenant ID. The value of the key is set as `X-Scope-OrgID` of HTTP header. It is useful to set Tenant ID dynamically.                                                                                                                                                                                                                                     |                   |
+| compress                        | Set payload compression mechanism. The only available option is gzip. Default = "", which means no compression.                                                                                                                                                                                                                                                                                                                 |                   |
+| workers                         | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output.                                                                                                                                                                                                                                                                                                            | `0`               |
```

Comment:
```suggestion
| `label_map_path`               | Specify the label map file path. The file defines how to extract labels from each record. More details in the Labels section. | _none_ |
| `structured_metadata`          | Optional, comma-separated list of `key=value` strings specifying structured metadata for the log line. Like the `labels` parameter, values can reference record keys using record accessors. See [Structured metadata](#structured-metadata) for more information. | _none_ |
| `structured_metadata_map_keys` | Optional, comma-separated list of record key strings specifying record values of type map, used to dynamically populate structured metadata for the log line. Values can only reference record keys using record accessors, which should reference map values. Each entry from the referenced map will be used to add an entry to the structured metadata. See [Structured metadata](#structured-metadata) for more information. | _none_ |
| `remove_keys`                  | Optional list of keys to remove. | _none_ |
| `drop_single_key`              | If set to true and after extracting labels only a single key remains, the log line sent to Loki will be the value of that key in `line_format`. If set to `raw` and the log line is a string, the log line will be sent unquoted. | `off` |
| `line_format`                  | Format to use when flattening the record to a log line. Valid values are `json` or `key_value`. If set to `json`, the log line sent to Loki will be the Fluent Bit record dumped as JSON. If set to `key_value`, the log line will be each item in the record concatenated together (separated by a single space) in the format. | `json` |
| `auto_kubernetes_labels`       | If set to true, it will add all Kubernetes labels to the Stream labels. | `off` |
| `tenant_id_key`                | Specify the name of the key from the original record that contains the Tenant ID. The value of the key is set as `X-Scope-OrgID` of HTTP header. It is useful to set Tenant ID dynamically. | _none_ |
| `compress`                     | Set payload compression mechanism. The only available option is gzip. Default is no value, which means no compression. | _none_ |
| `workers`                      | The number of [workers](../../administration/multithreading.md#outputs) to perform flush operations for this output. | `0` |
```